programs/eosc/main.cpp

```diff
@@ -959,15 +960,33 @@ int main( int argc, char** argv ) {
    benchmark_setup->add_option("owner", owner_key, localized("The owner key to use for account creation"))->required();
    benchmark_setup->add_option("active", active_key, localized("The active key to use for account creation"))->required();
    add_standard_transaction_options(benchmark_setup);
+
    benchmark_setup->set_callback([&]{
+      auto controlling_account_arg = fc::mutable_variant_object( "controlling_account", string("inita"));
```

Comment:
I think `inita` needs to be an argument since you prompt for owner and active keys. As it is now owner and active keys would have to be the keys of inita.